cachito/web/utils.py

```diff
@@ -0,0 +1,51 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+from flask import request, url_for
+
+
+def pagination_metadata(pagination_query):
+    """
+    Return a dictionary containing metadata about the paginated query.
+    This must be run as part of a Flask request.
+
+    :param pagination_query: flask_sqlalchemy.Pagination object
+    :return: a dictionary containing metadata about the paginated query
+    """
+
+    pagination_data = {
+        'page': pagination_query.page,
+        'per_page': pagination_query.per_page,
+        'total': pagination_query.total,
+        'pages': pagination_query.pages,
+        'first': url_for(
```

Comment:
Optional: For readability, it'd be nice if the keys were alphabetized

Reply:
Done!